alerta/auth/basic_ldap.py

```diff
@@ -17,12 +18,19 @@
 def login():
     # Retrieve required fields from client request
     try:
-        email = request.json.get('username', None) or request.json['email']
+        login = request.json.get('username', None) or request.json['email']
         password = request.json['password']
     except KeyError:
         raise ApiError("must supply 'username' and 'password'", 401)
 
-    username, domain = email.split('@')
+    email = ''
+    email_verified = False
+    if '\\' in login:
+        domain, username = login.split('\\')
+    else: 
+        username, domain = login.split('@')
+        email = login
+        email_verified = True
```

Comment:
Vey minor, but for this I would put ...
```
    email = ''
    email_verified = False
```
after the `if` so that it looked symmetrical.

That is...
```
    if '\\' in login:
        domain, username = login.split('\\')
        email = ''
        email_verified = False
    else: 
        username, domain = login.split('@')
        email = login
        email_verified = True
```